soh/soh/Enhancements/randomizer/savefile.cpp

```diff
@@ -219,9 +222,11 @@ extern "C" void Randomizer_InitSaveFile() {
     Flags_SetEventChkInf(EVENTCHKINF_DEKU_TREE_OPENED_MOUTH);
     Flags_SetInfTable(INFTABLE_SPOKE_TO_KAEPORA_IN_LAKE_HYLIA);
     Flags_SetEventChkInf(EVENTCHKINF_ENTERED_MASTER_SWORD_CHAMBER);
-    Flags_SetEventChkInf(EVENTCHKINF_PULLED_MASTER_SWORD_FROM_PEDESTAL);
+    // Moved to links pocket condition (awards light medallion)
+    // Flags_SetEventChkInf(EVENTCHKINF_PULLED_MASTER_SWORD_FROM_PEDESTAL);
```

Comment:
The “awards light medallion” is really just referring to the vanilla logic item behind this check.

This flag was previously used to skip the cutscene that plays the first time you pull MS (which is where you would get the light medallion in vanilla)

Now that cutscene is skipped in z_demo, and this flag is used in the ItemGiver mod to award the links pocket check.

This will allow us in the near future (after some logic tweaks) to award the links pocket check the first time you travel to adult. This should obviously be optional in the name of parity with OOTR, but IIRC the only reason the concept of “links pocket” exists is because of limitations we don’t have, we just inherited it.